src/server/index.ts

```diff
@@ -242,6 +243,11 @@ initDb()
 
     const port = 8080
     app.listen(port, () => logger.info(`Listening on port ${port}!`))
+
+    process.on('unhandledRejection', (error) => {
+      dogstatsd.increment(ERROR_UNHANLDED_REJECTION, 1, 1)
```

Comment:
nit, spelling error but otherwise lgtm!

Reply:
whoops, fixed in [f878e46](https://github.com/opengovsg/GoGovSG/pull/2080/commits/f878e46d01435526750ef1aee7277de119eb8b51)